xfuser/model_executor/models/base_model.py

```diff
@@ -84,14 +84,21 @@ def _wrap_layers(
                         f"{wrapper.__name__}"
                     )
                     if additional_args is not {}:
-                        setattr(
-                            module,
-                            subname,
-                            wrapper(
-                                submodule,
-                                **additional_args,
-                            ),
-                        )
+                        if "temporal_transformer_blocks" in name and subname == "attn1":
```

Comment:
Is there a more general way to modify this, such as adding an option like submodule_name_to_wrap to the upper level